src/nurse/management/commands/_notifications.py

```diff
@@ -9,13 +12,22 @@ def get_client():
 
 
 def notify():
+    soon_expiring_prescriptions = Prescription.objects.expiring_soon()
+    users_for_these_prescriptions = get_users_for_prescriptions(
+        soon_expiring_prescriptions
+    )
+    users_list = list(users_for_these_prescriptions.values())
```

Comment:
I wouldn't evaluate it early, but see if we could use it as a subquery passing it to the subscription_ids_for_users method

Reply:
done